api-test/src/main/resources/idRepository/uinrids/generateuin.yml

```
@@ -257,3 +257,7 @@ GenerateUIN:
       output: '{
   "status":"ACTIVATED"
 }'
+
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (4 > 0)

(empty-lines)